.github/workflows/workflow.yml

```diff
@@ -134,6 +134,9 @@ jobs:
       - uses: codecov/codecov-action@v3
         if: ${{ !steps.release-check.outputs.IS_RELEASE }}
         with:
-          file: coverage/coverage-final.json
+          files:
```

Comment:
NIT: in the future we can get it with a glob and then pass as variable here but something we can do in a follow up PR as well